lib/givt_app.dart

```diff
@@ -51,6 +55,14 @@ class GivtApp extends StatelessWidget {
           create: (BuildContext context) => ProfilesCubit(),
           lazy: false,
         ),
+        BlocProvider<CameraCubit>(
+          create: (BuildContext context) => CameraCubit(),
+          lazy: true,
+        ),
+        BlocProvider<OrganisationCubit>(
+          create: (BuildContext context) => OrganisationCubit(),
+          lazy: true,
+        ),
```

Comment:
Why do you put the cubits at a top level in the app? It's not considered a good practice, especially that you won't use the camera cubit in the whole app, but only in one page/scree

Reply:
I moved the CameraCubit Provider to the CameraScreen. 
But the OrganisationCubit Provider will be needed for multiple Screens where their only common ancestor is the GivtApp. 
We could make a widget wrapper for the donation flow and provide the OrgCubit there. @pavelpavelko  what do you think?

Reply:
we already discussed it today on the meeting but for visibility going to comment here as well: the downside of this idea is that there will be different instances of the OrgCubit provided for different screens.